src/rpc/evo.cpp

```diff
@@ -184,12 +184,14 @@ static RPCArg GetRpcArg(const std::string& strParamName)
                 "Platform P2P node ID, derived from P2P public key."}
         },
         {"platformP2PPort",
-            {"platformP2PPort", RPCArg::Type::NUM, RPCArg::Optional::NO,
-                "TCP port of Dash Platform peer-to-peer communication between nodes (network byte order)."}
+            {"platformP2PPort", RPCArg::Type::STR, RPCArg::Optional::NO,
```

Comment:
it's not more `platormP2PPort`; especially if it's string.

I'd suggest to create new field if you really need to return string for any reason; otherwise just return PORT.

Also I see further you requires platform address to has port only (not address), so, I'd suggest to don't change this one.
See:
```
                                  -8, "ProTx version disallows storing addresses in platformP2PPort (must specify port number only)")
```